app/services/api/endpointService.js

```diff
@@ -49,7 +49,7 @@ angular.module('portainer.services')
   service.createLocalEndpoint = function(name, URL, TLS, active) {
     var endpoint = {
       Name: 'local',
-      URL: 'unix:///var/run/docker.sock',
+      URL: 'npipe:\\\\.\\pipe\\docker_engine',
```

Comment:
Same thing here, I think that we'll need to have `createLocalLinuxEndpoint` and `createLocalWindowsEndpoint`.

Reply:
@deviantony what do you want to do here? Is there a way to extricate the server host OS with angular?